microcosm_flask/swagger/parameters/enum.py

```diff
@@ -38,9 +58,15 @@ def parse_type(self, field: Field) -> str:
         else:
             raise Exception(f"Cannot infer enum type for field: {field.name}")
 
-    def parse_enum_values(self, field: Field) -> Sequence:
+    def _parse_enum_values(self, field: Field) -> Sequence:
         enum = getattr(field, "enum", None)
         return [
             choice.value if field.by_value else choice.name  # type: ignore
             for choice in enum
         ]
+
+    def parse_enum_values(self, field: Field) -> Optional[Sequence]:
+        if not self.strict_enums:
+            return None
```

Comment:
Step 4: Output for an enum is an enum only with `strict_enums=True`.

Reply:
nit: Think code would be simpler if it matched your comment
```
if self.strict_enums:
     return self._parse_enum_value(field)
return None
```